sdk/instrumentation/opentelemetry-instrumentation-azure-sdk/test/internal/node/instrumentation.spec.ts

```diff
@@ -39,10 +39,8 @@ describe("instrumentation end-to-end tests", () => {
       assert.deepEqual(outer.status, { code: SpanStatusCode.OK });
 
       // Check instrumentationLibrary
-      assert.deepEqual(outer.instrumentationLibrary, {
-        name: tracingClientAttributes.packageName,
-        version: tracingClientAttributes.packageVersion,
-      });
+      assert.equal(outer.instrumentationLibrary.name, tracingClientAttributes.packageName);
```

Comment:
version 1.1.0 added an optional "schemaVersion" parameter which we do not use and can ignore (but deepEqual will complain about it)